src/Examples/Elmish Examples/Examples.Elmish.MusicPlayer/Songs.fs

```diff
@@ -29,3 +28,9 @@ module Songs =
                   name = name
                   path = path
                   createdAt = DateTime.Now })
+
+    let populateFromFolders(folders: IStorageFolder seq) =
+        folders
+        |> Seq.map _.Path.LocalPath
```

Comment:
@JaggerJo are we ok with using F# 8 syntax in the samples, or would you prefer sticking to an older version? (querying after I tried building on a system that only had .NET 7 installed and got build errors)

Reply:
I don't mind going with .NET 8.